install-local.sh

```diff
@@ -1,34 +1,40 @@
 usage() {
-    echo
-    echo "nsuite installer options:"
-    echo
-    echo "   arbor       : build Arbor"
-    echo "   neuron      : build NEURON"
-    echo "   coreneuron  : build CoreNEURON"
-    echo "   all         : build all simulators"
-    echo "   -e filename : source filename before building"
-    echo
-    echo "examples:"
-    echo
-    echo "install only Arbor:"
-    echo "$ install arbor"
-    echo
-    echo "install Arbor, NEURON and CoreNEURON:"
-    echo "$ install all"
-    echo
-    echo "install NEURON using environment configured in config.sh:"
-    echo "$ install neuron -e config.sh"
-    echo
+    cat <<'_end_'
+Usage: install-local.sh [-e SCRIPT] [--prefix=PATH] TARGET
+
+Setup NSuite framework, build and install simulators, benchmarks,
+and validation tests.
+
+Options:
+    -e SCRIPT       Source SCRIPT before building.
+    --prefix=PATH   Use PATH as base for install and other run-time
+                    working directories.
+
+TARGET is one of:
```

Comment:
`is one of` doesn't make it clear that you can list more than one target

Reply:
I've changed the first line of usage to:
```
Usage: install-local.sh [--env=SCRIPT] [--prefix=PATH] TARGET [TARGET...]
```